### PR TITLE
fix(collaboration): added new collaboration documents

### PR DESCRIPTION
**Pull request by homelessbirds**
NR-65539
NR-65922

### CODE REVIEW

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+import collaboration_discussion_board_new from 'images/collaboration-discussion-board-new.png'
+
+import green_dot from 'images/green_dot.png'
+
+<Callout variant="important">
+The functionality described below is currently in preview. If you don't yet have access but would like to test the commenting and discussion functionality [please sign up here](https://one.newrelic.com/discussions).
+</Callout>
+
+We're making it easier to collaborate with your teammates by building discussion and commenting tools into New Relic. You can now add comments on any page. Our integrations with Slack, MS Teams (coming soon), and email let you share links and screenshots, and communicate with your teammates directly from the New Relic UI without switching applications. Our collaboration tools allow you to identify problems and find solutions in one place. Moving the discussion closer to your data streamlines your workflows, improves efficiency, and reduces MTTR.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:20]: The way I read the last part of the first sentence, it read as "building discussion" and "commenting tools". I think the difference in discussion and commenting can make it a little harder to parse. Maybe something like "We're making it easier to collaborate with your teammates by integrating discussion tools into New Relic, allowing you to add comments on any page." Maybe not "integrating" but something along those lines.

Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:20]: Not sure there needs to be a comma between "links and screenshots" and "and communicate"

Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:20]: Really small thing but I think you could probably merge the last two sentences with a comma and changing "streamlines", "improves", and "reduces" to "and streamlining", "improving", and "reducing". This is just a super small thing though feel free to ignore.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+<Video
+ type="wistia"
+ id="m4b6y854wi"
+/>
+
+We've integrated discussion tools directly into New Relic so you can: 
+
+* **Start discussions in context at the source.** When you comment on a dashboard or an error in New Relic, that comment stays with that dashboard or error.
+* **Sync discussions with Slack.**  With our Slack integration you can share links, screenshots, and comments to Slack in seconds. All teammates  can reply from Slack and discussions stay in sync, no matter where teammates contribute. [Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:32]: Should there be a comma after "With our Slack integration"?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+* **Create shared knowledge and history.** Collaborating within the New Relic platform will allow your team to build a searchable body of shared knowledge.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:33]: I think "will allow" can just be "allows", feels stronger

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+## Comment on any page [#comment-on-any-page]
+
+On the global navigation, there's a new “add comment” icon on most pages. A green dot on the icon means there is actually an open discussion on the page. A gray dot means there are closed discussions that can perhaps answer new questions or help resolve current issues.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:36]: I think the last two sentences should be consolidated and tweaked a bit, something like "A green dot on the icon means there's an open discussion on the page, while a gray dot indicates a closed discussion that may have information to help resolve a current issue." What do you think?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+On the global navigation, there's a new “add comment” icon on most pages. A green dot on the icon means there is actually an open discussion on the page. A gray dot means there are closed discussions that can perhaps answer new questions or help resolve current issues. 
+
+<img
+  title="Comment on any page "
+  alt="A screenshot of the UI that depicts commenting on a specific page."
+  src={green_dot}
+/>
+
+If you want to ask a question, start a discussion, or share a permalink or screenshot to Slack, click the icon to open the comment panel. Type your comment and start a discussion on the page you're looking at. When you comment on a dashboard or an error in New Relic, that comment stays with your data. You don't need to spend time repeating information that can be seen at a glance, and everybody knows where to find an active, solution-oriented conversation.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:44]: The second sentence has the feeling of being kind of procedural. What do you think about removing it and baking it into the third sentence, something like "When you enter a comment on a dashboard or an error..."

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+You may notice the “add comment” icon is disabled on some pages. By design, in a multi-account organization you can't discuss pages that either don't have data associated with a specific account. You also won't be able to discuss on pages that present data belonging to multiple accounts in a single view. Simply selecting a specific account or clicking through to a specific item or entity will usually show the “add comment” icon.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:46]: Is there something missing from the sentence starting with "By design" around the "either?"

Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:46]: The "present" can be read incorrectly (present being indicative of time instead of indicative of an action), what do you think about changing it to just "show" or something like that?

Comment (jeff-colucci) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:46]: I think you can cut Simply from the last sentence

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+## At-mention your teammates and notify by email [#at-mention-your-teammates]
+
+Like other collaboration tools, when you at-mention (@) your teammates in a comment, they'll be notified by email. If you at-mention a teammate that isn't part of the preview or isn't using the new redesign, they won't have access to the discussion on New Relic. 
+
+## Share links and screenshots with Slack [#sync-with-slack]
+
+Whenever you start a new discussion, you can sync it with Slack so you can collaborate with all your teammates, even those that aren't on New Relic.
+
+Once the Slack app is configured and you are authenticated, select the Slack channel where you want your discussion synced. If you regularly use the same Slack channel it'll be prefilled so no extra clicks needed. From Slack, any teammate, even if they are not a New Relic user, can reply and their comment will be brought back into the thread on New Relic. Likewise subsequent replies added to the thread on New Relic will also be added to the thread on Slack.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:56]: For the second sentence I think there should be a comma after "Slack channel". Also in this sentence, there should be an "are" between "extra" and "clicks"

Comment (jeff-colucci) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:56]: There are a lot of commas in the third sentence that can make it kind of hard to parse. What do you think about something like "Any teammate can reply in Slack, and their comment will be integrated into the thread on New Relic even if they are not a New Relic user." or something along those lines?

Comment (jeff-colucci) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:56]: I think you can cut Likewise from the last sentence.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+[Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:58]: Can this be put into the previous paragraph?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+
+## Built-in screenshot sharing [#built-in-screenshots]
+
+With a new integrated screenshot selection tool you can attach screenshots to your comments. Click the “screenshot” icon, select the portion of the screen you want, and submit your comment.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:62]: I think "our new" instead of "a new" works a bit better for the screenshot tool. You can start the second sentence with "Just" if you want to soften the tone a bit as well.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+[Learn more about the Slack integration](https://docs-preview.newrelic.com/docs/collaboration-slack-integration).
+
+## Built-in screenshot sharing [#built-in-screenshots]
+
+With a new integrated screenshot selection tool you can attach screenshots to your comments. Click the “screenshot” icon, select the portion of the screen you want, and submit your comment. 
+
+## Close and re-open discussions [#close-reopen]
+
+Once a question is answered or issue is resolved you can mark a discussion as “Closed” via the more actions menu on the discussion thread. Similar to a Jira ticket your team can still add comments to the discussion. This will not automatically re-open a discussion but if you need to you can manually do so the same way as closing.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:66]: I think there should be an "an" before "issue" in "issue is resolved", which should also have a comma after the "resolved". I'm also not 100% sure on the styling yet but should there be something indicating "more actions" as a thing the user interacts with?

Comment (jeff-colucci) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:66]: I think the second sentence should be flipped around: Your team can still add comments to the discussion similar to a Jira ticket.

Comment (jeff-colucci) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:66]: I think there should be a comma after "discussion" in the third sentence, and you can also probably cut out "if you need to you". Also the "closing" at the end should be followed by what they are closing.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+<img
+  title="Close and reopen discussions "
+  alt="A screenshot of the UI that depicts closing discussions."
+  src={close_and_reopen_discussions}
+/>
+
+## Follow and unfollow discussions [#follow-unfollow]
+
+When you add a comment to a discussion or if you're at-mention you'll automatically be set to follow the discussion. This will be indicated by the bell icon on the discussion and on the discussions board. Following a discussion means you'll get email notifications and the Discussions board UMI (see next section) will increment when new comments are added to a discussion. If you want to unfollow a discussion that's also available via the more actions menu on the discussion (see screenshot in doc above).  Likewise, if there is a discussion you are interested in that you're not following but would like to, you can manually follow the same way.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:76]: Should say "if you're at-mentioned" and have a comma following the "at mention".

Comment (jeff-colucci) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:76]: The second two sentences are a bit unclear to me, is it saying "If you want to unfollow a discussion, you can do so via the more actions menu"? I think the second of the two sentences is only unclear because the first one is, but like before you can also cut out the Likewise.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+## The discussions board [#discussions-board]
+
+<img
+  title="Discussions board"
+  alt="A screenshot of the UI that depiects the discussions board."
+  src={collaboration_discussion_board_new}
+/>
+
+Go to the discussions board to see a complete history of the discussion threads in your organization. On the left nav, the UMI on the Discussions link indicates discussion threads you are following that have unread comments.. On the Discussions page, click a discussion thread to jump to the UI where that conversation is happening.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:86]: Double period after comments.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+We know that your team uses a wide variety of tools to communicate but when discussions are spread across platforms and teams, it is easy to lose institutional knowledge. You can now use the discussion board to find active discussions as well as search the archive of shared knowledge. Our discussion board helps you find answers to questions and research how issues and problems were solved in the past.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:88]: I like feel and tone of this paragraph a lot. I think there should be a comma after communicate and it is perfect.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+All users in an account, and this applies to specific accounts in a multi-account organization, can see all discussions in that account. The Discussions page shows all discussions tied to data in accounts you are in, but not discussions tied to accounts you are not in.  (Keep in mind this applies to replies synced to New Relic from Slack, which may be composed in a small private channel on Slack, but will be viewable by all New Relic users in the account on New Relic.)
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:90]: This feels more like it is being spoken instead of read, if that makes sense. What do you think of this: "All users associated with an account can see the discussions that account has, whether it is a specific account or one of many in a multi-account organization. The discussions page shows all discussions tied to your account's data, but not discussions tied to accounts that aren't yours. This applies to replies synced from small or private channels in Slack which may become viewable by all users of the New Relic account."

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+
+import slack_auth from 'images/slack_auth.png'
+
+import slack_synced from 'images/slack_synced.png'
+
+import slack_channel from 'images/slack_channel.png'
+
+While not everyone you work with is a New Relic user, most of your teammates are probably on Slack. Now you can discuss New Relic data with everyone and keep all insights synced between Slack and New Relic discussions.
+
+Unlike many Slack integrations that just share an update to Slack in one-direction, conversation threads on New Relic are synced in both directions with threads on Slack. When you share a question, observation, or request for assistance from New Relic to Slack, your teammates on Slack—whether they are New Relic users themselves or not—can reply to the post on Slack and these replies will be brought back into the thread on New Relic. Likewise, replies added to the thread on New Relic will be added to the thread on Slack. This back and forth sync lets you include more people in the conversation, capture the discussion along with the context in New Relic, use data to inform decisions, and build institutional knowledge. All at the same time.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx:21]: Really like the style of the first two paragraphs. You may want to merge the last sentence in with the previous one but overall this is super-good

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+
+## Installation [#install]
+
+The New Relic discussions Slack app is not yet in the Slack App Directory. Depending on your Slack workspace's **App Management Settings**, you may need the assistance of your Slack administrator for installation. 
+
+If set, your Slack workspace admin may need to “Turn off “Only allow apps from the Slack App Directory,”, and likewise turn on “Allow members to request approval for apps.”. Then you can proceed with the authentication instructions below. Following those instructions will generate a request for an admin to approve the application after which, you can install the Slack app, and any settings temporarily changed can be restored.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx:27]: "Once" instead of "if" maybe? That way we're kind of being more enticing with the prospect.

Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx:27]: I think you can move the comma that's after "after which" to after "application"

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+
+<img
+  title="Slack app management "
+  alt="A screenshot of the new home page UI."
+  src={slack_app_management}
+/>
+
+## Authentication [#authenticate]
+
+To use the New Relic Discussions Slack application you'll need your New Relic admin (or teammate with the Applied Intelligence Destination Create capability) to be the first to authenticate with Slack. This can be initiated by clicking on the “Sync to Slack” icon which will send you to slack.com to login to the workspace you'd like to use.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx:37]: Do we bold or put parenthesis around selectable buttons/options in the UI? I'm actually not sure myself so I want to make sure I'm not missing anything.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+
+The first comment in a thread synced to Slack will be added to the channel selected on New Relic. The Slack post will include an “Open in New Relic” button providing quick navigation to the page and discussion on [one.newrelic.com](one.newrelic.com).
+
+<img
+  title="Slack reply thread"
+  alt="A screenshot of a Slack reply thread."
+  src={post_on_slack}
+/>
+
+The post will also include a message indicating the thread is synced to New Relic and that replies added to the thread on Slack will appear on New Relic as well.  Please note this means that regardless of the Slack channel's member list **replies will be visible by everyone in your New Relic organization on newrelic.com**.
```
Comment (jeff-colucci) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx:81]: For the last sentence, what do you think about switching it around again to "Please note that this means that replies will be visible by everyone in your New Relic organization on newrelic.com regardless of the Slack channel's member list." ?

**Review by jeff-colucci — COMMENTED** [review]
I found a few things, as always any suggestions you want to ignore please feel free to do so! Let me know when you want me to do another review for approval.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:20]: I like the way you phrased that, thanks @jeff-colucci

Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:20]: thanks!

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx (hunk 6 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:46]: yes! good catch!

Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:46]: thanks :)

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx (hunk 11 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:76]: thanks!

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx (hunk 13 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaborate-with-teammates.mdx:88]: :)

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx (hunk 17 above)
Comment (homelessbirds) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/collaborate/collaboration-slack-integration.mdx:37]: I actually don't know this either!

**Review by jeff-colucci — DISMISSED** [review]
I just noticed a few tiny things, other than that looks great!
